### DISCUSSION TITLE
Welcome to theartofeducation.github.io Discussions!

### DISCUSSION THREAD

**Discussion by theantichris** [theartofeducation/theartofeducation.github.io#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jmosley5** [comment]
Judy Mosley here! QA Engineer with The Art of Education University. Ready to engage and help however I can 😄
